### ISSUE TITLE
can''t compile the driver

### ISSUE THREAD

**Issue by grap12** [cheat-engine/cheat-engine#2955]
Im getting error windows7 is not compatible when compiling the driver Im using vs 2022 and latest wdk and sdk on windows 11

**Comment by fade2gray** [comment]
Project -> Properties -> Driver settings -> General

Make sure you change Target OS Version to 'Windows 10 or higher'.

![TargetOS](https://github.com/user-attachments/assets/3d0fbe31-a3c7-4016-b8df-2e3fe9024618)

**Comment by grap12** [comment]
> Project -> Properties -> Driver settings -> General
> 
> Make sure you change Target OS Version to 'Windows 10 or higher' in the dropdown selector.
> 
> ![TargetOS](https://private-user-images.githubusercontent.com/1332009/348738639-3d0fbe31-a3c7-4016-b8df-2e3fe9024618.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjExMjk4OTUsIm5iZiI6MTcyMTEyOTU5NSwicGF0aCI6Ii8xMzMyMDA5LzM0ODczODYzOS0zZDBmYmUzMS1hM2M3LTQwMTYtYjhkZi0yZTNmZTkwMjQ2MTgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTZUMTEzMzE1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9N2I0YTk3MTcyMTcyYzE1ZmU5YTE4MzkwZjI3OWY4ZjU4NGJiYzMwNzkzNDY5ZTJkMTQwZGJkMzcwMzQxYzYyMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.YZ9Q8zihUFK1JNLDlGsYITSLEulQ8NaA6q8qrTFjMAM)

![image](https://github.com/user-attachments/assets/af015cf5-625a-4cfe-b002-4493368079b6)

even with this still does not work this a problem with latest swdk and wdk can't support any driver that still support windows7

**Comment by fade2gray** [comment]
Personally, I use VS2019 following a guide by Baseult.

Search for "YouTube Baseult Undetected Cheat Engine" - he also provides a link in the description to an easier to follow forum guide.

**Comment by grap12** [comment]
> Personally, I use VS2019 following a guide by Baseult.
> 
> Search for "YouTube Baseult Undetected Cheat Engine" - he also provides a link in the description to an easier to follow forum guide.

do u have windows 11 because I find on this link https://stackoverflow.com/questions/73450804/why-does-the-latest-wdk-not-support-windows-7-by-default that the latest wdk does not support any driver that support windows 7

**Comment by fade2gray** [comment]
Yes, I use Windows 11. The guide works for Win11 as well as Win10 - make sure to use the Win10 SDK and WDK as shown in the guide, do not use the Win11 ones!!!

**Comment by grap12** [comment]
> Yes, I use Windows 11. The guide works for Win11 as well as Win10 - make sure to use the Win10 SDK and WDK as shown in the guide, do not use the Win11 ones!!!

I do what u said I get 
Error	C1189	#error:  "No Target Architecture"

![image](https://github.com/user-attachments/assets/f1c5d074-306b-485c-96ff-54614a866004)
